src/components/behandlerdialog/skrivtilbehandler/SkrivTilBehandlerSkjema.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import { SkjemaFeiloppsummering } from "@/components/SkjemaFeiloppsummering";
+import { FlexRow } from "@/components/Layout";
+import { Button } from "@navikt/ds-react";
+import { Form } from "react-final-form";
+import { VelgBehandler } from "@/components/behandlerdialog/skrivtilbehandler/VelgBehandler";
+import { BehandlerDTO } from "@/data/behandler/BehandlerDTO";
+import styled from "styled-components";
+import { MeldingTekstfelt } from "@/components/behandlerdialog/skrivtilbehandler/MeldingTekstfelt";
+import {
+  behandlerRefValidationErrors,
+  validerSkjemaTekster,
+} from "@/utils/valideringUtils";
+import { useFeilUtbedret } from "@/hooks/useFeilUtbedret";
+
+const texts = {
+  knappTekst: "Send til behandler",
+};
+
+const validationTexts = {
+  missingMeldingTekst: "Innholdet i meldingen er tomt", // TODO: se nærmere på teksten
+};
+
+const SkrivTilBehandlerFormWrapper = styled.div`
+  margin-top: 1.5em;
+`;
+
+const StyledForm = styled.form`
+  > * {
+    &:not(:last-child) {
+      margin-bottom: 1.5em;
+    }
+  }
+`;
+
+interface SkrivTilBehandlerSkjemaValues {
```

Comment:
Du kan fjerne `meldingTekst` her og heller extende `SkrivTilBehandlerSkjemaTekster` så slipper man definere `meldingTekst` to steder.

Reply:
Jepp, jeg endret også navn fra `SkrivTilBehandlerSkjemaTekster` til `SkrivTilBehandlerSkjemaFritekstfelter`. Synes tekster alene kunne misforståes med feks de `const texts = {}` som vi pleier å ha.